src/_tests_/App.test.js

```diff
@@ -44,5 +52,57 @@ describe('App', () => {
     it('should match the snapshot', () => {
       expect(wrapper).toMatchSnapshot();
     });
+
+    it('should match the snapshot when isLoading is true', () => {
+      wrapper = shallow(<App {...mockProps} isLoading={true} />);
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should match the snapshot when there is an error', () => {
+      wrapper = shallow(<App {...mockProps} error={'Error fetching data'} />);
```

Comment:
really like how you defined mockProps as a variable and using the spread operator to pass the props to App